checklists/bronze.md

```diff
@@ -5,13 +5,16 @@
 
 ### Documentation
 
-- [ ] Brief description of what program does is listed.
-- [ ] An example of its usage is shown.
-  - typical usage
-- [ ] Description of required input parameters.
-  - Reasonable description (i.e. "Nifti of brain mask in MNI" vs. "An image file")
-- [ ] Description of output(s).
-
+- [ ] README provides link to documentation
+- [ ] Documentation is up-to-date
+- [ ] Brief description of what program does is listed
+- [ ] Typical intended usage is described
```

Comment:
Again, I'd think of this in the context of the landing page. Maybe what we want is to nest criteria here? Some of these should certainly be accessible from the high-level landing page, and others in the more dense documentation. If we make sub-lists for each, it could help reviewers and make sure that the pieces are easy to find for consumers.